frontend/src/components/accountdetails/AccountDetailsPage.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import AccountDetailsTable from './AccountDetailsTable';
+
+export default class AccountDetailsPage extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.onClickDeleteAccount = this.onClickDeleteAccount.bind(this);
+    this.onDepositClick = this.onDepositClick.bind(this);
+    this.onWithdrawClick = this.onWithdrawClick.bind(this);
+    this.onClickAccountHistory = this.onClickAccountHistory.bind(this);
+    this.accountFilter = this.accountFilter.bind(this);
+  }
+
+  render() {
+    return (
+      <div className="jumbotron">
+        <h2>Account Details</h2>
+        {this.accountFilter()}
+        <button className="btn btn-default btn-md" onClick={this.onDepositClick}>Deposit</button>
+        <button className="btn btn-default btn-md" onClick={this.onWithdrawClick}>Withdraw</button>     
+        <button className="btn btn-default btn-md" onClick={this.onClickDeleteAccount}>Delete</button>
+        <button className="btn btn-default btn-md" onClick={this.onClickAccountHistory}>Transactions History</button>
+        <div className='div--back-button'>
+          <Link to='/accounts'>
+            <button className="btn btn-default btn-md">Back</button>
+          </Link>
+        </div>
+      </div>
+    );
+  }
+  
+  accountFilter() {
+    let targetAccount = undefined;
+    let targetId = this.props.match.params.id;
+    this.props.accounts.forEach(function(account) {
```

Comment:
I suppose the state here should be for the selectedAccount. 

e.g when user enters /accounts/34 (or goes there from the table) then the component sends loading request to the server for retrieving of the account with id 34 and on success the reducer will map response to the selectedAccount property in the state of the store.